client/components/seo/facebook-preview/index.jsx

```diff
@@ -54,7 +58,7 @@ export class FacebookPreview extends PureComponent {
 							{ facebookDescription( description || '' ) }
 						</div>
 						<div className="facebook-preview__url">
-							{ baseDomain( url ) }
+							{ baseDomain( url ) + authorInfo( author ) }
```

Comment:
this kind of operation is often done with a `join()` - you could reuse `compact()` to rid empty strings…

```js
{ compact( [ baseDomain( url ), author ] ).join( ' | ' ) }
```